src/pages/VHomepage.vue

```diff
@@ -0,0 +1,50 @@
+<script setup lang="ts">
+import { MaButton } from '@mobileaction/action-kit';
+</script>
+
+<template>
+    <div class="homepage-container">
+        <div class="content-center">
+            <h1 class="main-title">Welcome to Onboarding Progress</h1>
+            <p class="description">
+                You can see your first task details via the link below. 
+                Please continue with these steps while coding.
+            </p>
+            <div class="button-section">
+              <a href="https://mobileaction.atlassian.net/l/cp/7fw8MroM" target="_blank">
+                <MaButton 
```

Comment:
Please use kebab case on component usages its a better practice for vue
```suggestion
                <ma-button 
```